dashboards/kibana/kibana-prometheus.json

```diff
@@ -75,25 +91,31 @@
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "rate(kibana_millis_uptime[5m])"
+                  "prometheusQuery": "rate(kibana_millis_uptime[5m])",
```

Comment:
I don't think this should have `rate(` applied to it.

I believe uptime is meant to keep going up over time (until kibana restarts, then it would go back to zero), but with `rate`, this becomes something like "increase in uptime per second" which means it will always plateau at 1000ms = 1 second per second.